tests/unit/ui/components/TestRating.js

```diff
@@ -101,6 +101,18 @@ describe('ui/components/Rating', () => {
     });
   });
 
+  it('renders selected stars corresponding to rounds ratings to nearest 0.5 multiple', () => {
```

Comment:
I think this needs a test for something like `4.8` average to render as 5 stars.